### PR TITLE
Default to reverse order for past events endpoint

### PR DESCRIPTION
**Pull request by sumeet-bansal**
By request but also it just makes sense.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
     "typedi": "^0.8.0",
     "typeorm": "^0.2.25",
     "typeorm-typedi-extensions": "^0.2.3",
-    "typescript": "^3.9.5",
+    "typescript": "^4.0.0",
```
Comment (sumeet-bansal) [inline package.json:64]: This is to pick up the `??=` operator (see [here](https://github.com/microsoft/TypeScript/issues/37255)).

**Review by shravanhariharan2 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (trulyronak) [reply package.json:64]: this is god tier ty for sharing

File: repositories/EventRepository.ts [hunk 2]
```diff
@@ -49,7 +49,7 @@ export class EventRepository extends BaseRepository<EventModel> {
     let query = this.repository.createQueryBuilder()
       .skip(options.offset)
       .take(options.limit)
-      .orderBy('start', 'ASC');
+      .orderBy('start', options.reverse ? 'DESC' : 'ASC');
```
Comment (trulyronak) [inline repositories/EventRepository.ts:52]: while this makes more sense, honestly lets just default to `ASC` to avoid a synchronized frontend push

**Review by trulyronak — APPROVED** [review]
lgtm, but i dont think we should make a breaking change / rely on frontend to change in sync (but idrc)

File: repositories/EventRepository.ts (hunk 2 above)
Comment (trulyronak) [reply repositories/EventRepository.ts:52]: instead for [402](https://github.com/acmucsd/membership-portal-ui/pull/402) the code change would simply be to pass in the `reverse` option in the request (so we don't waste time doing the reverse, but we avoid making a breaking change)

File: package.json (hunk 1 above)
Comment (sumeet-bansal) [reply package.json:64]: @trulyronak check out the TypeScript roadmap ([here](https://github.com/Microsoft/TypeScript/wiki/Roadmap)) for all sorts of nifty things.

File: repositories/EventRepository.ts (hunk 2 above)
Comment (sumeet-bansal) [reply repositories/EventRepository.ts:52]: The ternary does default to `ASC` if `options.reverse` isn't set. And yeah, we should just pass it in as part of the request but I think it makes more sense for the past events endpoint to default to `DESC` anyways.

**Pull request merged**